students/ericstreit/session03/list_lab.py

```diff
@@ -0,0 +1,79 @@
+#Lesson03
+#String Exercises ##
+#
+#!/usr/bin/env python3
+#define variables
+mylist = ["Apples", "Pears", "Oranges", "Peaches"]
+#define function
+def myfunc(n):
+    """Update the DocString"""
+    pass
+#series 1
+print("The list currently contains: ", mylist)
+additem = input("Pleaes enter another item: ")
+mylist.append(additem)
+print("The list currently now contains: ", mylist)
+l = len(mylist)
+listitem = int(input("Enter a list item from 1 to {} and I will show it to you: ".format(l)))
+print(mylist[(listitem-1)])
+print()
+pause = input("OK, let's add Kiwis to the list!")
+newlist = ["Kiwis"]
+mylist = newlist + mylist
+print(mylist)
+pause = input("Now adding Mangos to the list!")
+mylist.insert(0,"Mangos")
+print(mylist)
+pause = input("Now displaying all items that begin with the letter 'P'")
+for i in mylist:
```

Comment:
I tend to use more meaning full names that "i" -- i is good for indexes, but when the loop is more than two lines, it helps have a meaning ful name:

```
for fruit in mylist:
   ...
```